src/System.Reflection.Metadata/src/System/Reflection/Metadata/TypeSystem/TypeDefinition.cs

```diff
@@ -52,6 +52,11 @@ public TypeAttributes Attributes
             }
         }
 
+        /// <summary>
+        /// Is the type is nested.
```

Comment:
nit: to make it aligned with other MSDN docs `Indicates whether this is the nested type.`

Reply:
done.